app/policies/group_member_policy.rb

```diff
@@ -3,25 +3,16 @@ class GroupMemberPolicy < ApplicationPolicy
 
   def update?
     required_permission = record.pleb? ? :members : :leaders
-    has_group_permission?(required_permission)
+    is_owner? || has_group_permission?(required_permission)
```

Comment:
Will this allow users to give themselves permissions that they shouldn't be able to? @NuckChorris